blockbase/inc/disable-site-editor.php

```diff
@@ -3,11 +3,25 @@
 /**
  * Adds a setting to the Gutenberg experiments page to disable the Site Editor.
  */
-function add_disable_site_editor_setting() {
+function blockbase_disable_site_editor() {
 	if ( ! is_readable( get_stylesheet_directory() . '/block-templates/index.html' ) ) {
 		return;
 	}
 
+	if ( is_admin() ) {
+		add_action( 'admin_init', 'blockbase_add_settings_field' );
+	}
+
+	if ( get_option( 'gutenberg-experiments' ) ) {
+		if ( array_key_exists( 'universal-theme-disable-site-editor', get_option( 'gutenberg-experiments' ) ) ) {
+			add_action( 'admin_init', 'blockbase_readd_legacy_admin_links' );
+			add_action( 'admin_init', 'blockbase_remove_site_editor_admin_link' );
+			add_action( 'admin_bar_menu', 'blockbase_remove_site_editor_link', 50 );
```

Comment:
This is the right hook, but it's not firing because the conditional is not being met. 

When I move this action outside the top `if ( get_option( 'gutenberg-experiments') )` conditional, then the site editor link from the admin bar is correctly removed.